src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,12 @@
+export default function ReviewCard({title, owner, votes,}) {
+    return(
+        <article className="ReviewBox">
```

Comment:
the css looks good, I know its designed for desktop but dont forget later on to make it responsive :)